jwst/saturation/saturation.py

```diff
@@ -65,8 +67,9 @@ def flag_saturation(input_model, ref_model, n_pix_grow_sat):
         sat_dq = ref_sub_model.dq.copy()
         ref_sub_model.close()
 
-    gdq_new, pdq_new = flag_saturated_pixels(data, gdq, pdq, sat_thresh,
-                                             sat_dq, ATOD_LIMIT, dqflags.pixel, n_pix_grow_sat)
+    gdq_new, pdq_new, _ = flag_saturated_pixels(
```

Comment:
i made a comment about this in stcal too, but i'm confused about whats going on here. the zframe is passed in, and is it corrected always if it exists? i don't think its super time intensive but wouldn't it technically save some time if it was only processed when it needs to be used (when all other groups are saturated and zeroframe is not)? im also confused why a _ is being returned here, shouldnt the corrected frame be returned?

Reply:
This was a mistake.  I replaced `_` with `zframe` and use it properly now.